src/main/java/org/jvnet/hudson/plugins/monitoring/PluginImpl.java

```diff
@@ -118,6 +118,10 @@ private boolean isParameterUndefined(String key) {
 	@Override
 	public void postInitialize() throws Exception {
 		super.postInitialize();
+                if (filter == null) {
+                    throw new Exception("Post-initialization hook has been called before the plugin start. "
+                        +"Filters are not available");
```

Comment:
The string was too long according to the coding style. It will be converted to a single constant in any case